src/sdg/core/models/logius.py

```diff
@@ -189,6 +190,7 @@ def generate_initial_data(self, catalog: ProductenCatalogus):
     class Meta:
         verbose_name = _("uniforme productnaam")
         verbose_name_plural = _("uniforme productnamen")
+        unique_together = (("upn_uri", "grondslag"),)
```

Comment:
Here it's better use the constraints option instead:

```
class Meta:
        constraints = [
            models.UniqueConstraint(fields=["upn_uri", "volume_number"], name="unique_uri_and_grondslag")
        ]
```

Ref: https://docs.djangoproject.com/en/3.2/ref/models/options/#unique-together

Reply:
Altho I prefer readability from the "old way", the threat of deprecation is indeed reason to use the new way.